src/platform/user/profile/vap-svc/containers/AddressValidationView.jsx

```diff
@@ -248,41 +252,83 @@ class AddressValidationView extends React.Component {
       <div key={id} className="address-validation-container">
-        {isFirstOptionOrEnabled &&
-          hasConfirmedSuggestions && (
-            <input
-              type="radio"
-              id={id}
-              onChange={() => {
-                this.onChangeSelectedAddress(address, id);
-              }}
-              checked={selectedAddressId === id}
-            />
-          )}
-        {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-        <label
-          htmlFor={id}
-          className="vads-u-margin-top--2 vads-u-display--flex vads-u-align-items--center"
-        >
-          <div className="vads-u-display--flex vads-u-flex-direction--column vads-u-padding-bottom--0p5">
-            <span
-              className="dd-privacy-hidden"
-              data-dd-action-name="street address"
-            >
-              {street}
-            </span>
-            <span
-              className="dd-privacy-hidden"
-              data-dd-action-name="city, state and zip code"
+        {isFirstOptionOrEnabled && hasConfirmedSuggestions ? (
+          <VaRadio
+            data-testid="va-radio-label"
+            label={
+              id === 'userEntered' ? 'Address you entered:' : puralizedAddress
+            }
+            labelHeaderLevel={5}
+            onVaValueChange={event => {
+              this.onChangeSelectedAddress(address, event.detail.value);
+            }}
+            className={
+              id === 'userEntered'
+                ? 'vads-u-margin-top--12'
+                : 'vads-u-margin-top--2'
+            }
+          >
+            {id === 'userEntered' ? (
+              <va-radio-option
+                data-testid="userEnteredAddressOption"
+                style={{ whiteSpace: 'pre-line' }}
+                key="userAddress"
+                name="addressGroup"
+                label={formatDisplayAddressInRadio(address)}
+                description={(street, cityStateZip, country)}
+                value="userEntered"
+                checked={selectedAddressId === id}
+              />
+            ) : (
+              confirmedSuggestions.map((suggestedAddress, index) => (
+                <va-radio-option
+                  data-testid="suggestedAddressOption"
+                  style={{ whiteSpace: 'pre-line' }}
+                  key="suggestedAddress"
+                  name="addressGroup"
+                  label={formatDisplayAddressInRadio(suggestedAddress)}
+                  description={(street, cityStateZip, country)}
+                  value={index}
+                  checked={selectedAddressId === index.toString()}
+                />
+              ))
+            )}
+          </VaRadio>
+        ) : (
+          <>
+            <h5 className="vads-u-margin-top--3 vads-u-padding-top--0">
+              Address you entered:
+            </h5>
+            {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+            <label
```

Comment:
I'm curious as to why you're using a label here without an associated form element that in turn necessitates disabling this eslint rule. Is there an alternative element you could wrap this in and if needed apply the appropriate classes / styles to that would achieve the same visual end result without the potential accessibility issue of a label without a form element?